### PR TITLE
[UNR-2445][MS] Doc changes for server travel.

### PR DESCRIPTION
**Pull request by MatthewSandfordImprobable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### ServerTravel doc changes

Not sure if we actually want to outline how to server travel as it is already an Unreal thing.

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them. 
@ElleEss

### CODE REVIEW

File: SpatialGDK/Documentation/unreal-features-support.md [hunk 1]
```diff
@@ -319,8 +319,8 @@ Support of Unreal features with the GDK in a single server-worker configuration:
  <tr>
     <td rowspan="3"><a href="https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling"><b>Travel<b></a></td>
     <td>Server Travel</td>
-    <td class="unplanned"></td>
-    <td>Server Travel is on the <a href="https://github.com/spatialos/UnrealGDK/projects/1#card-22461878"> roadmap</a> but is not currently supported.</td>
+    <td class="supported"></td>
+    <td>Not supported when used along side zoning or off-loading</td>
```
Comment (oblm) [inline SpatialGDK/Documentation/unreal-features-support.md]:
```suggestion
    <td>Not supported when used in conjunction with zoning or offloading</td>
```

File: SpatialGDK/Documentation/content/map-travel.md [hunk 2]
```diff
@@ -44,7 +44,10 @@ PlayerController->ClientTravel(TravelURL.ToString(), TRAVEL_Absolute, false /*bS
 ```
 
 ## UWorld::ServerTravel
-> Warning: `ServerTravel` is on the [roadmap](https://github.com/spatialos/UnrealGDK/projects/1#card-22461878) but is not currently supported.
+
+Server travel is supported on single server worker deployments. For testing in PIE, ensure that you have "Use Single Process" disabled in "Editor Preferences/Level Editor/Play/Multiplayer Options" and set "Editor Multiplayer Mode" to "Play As Client".
```
Comment (oblm) [inline SpatialGDK/Documentation/content/map-travel.md]:
```suggestion
Server travel is supported in single server-worker deployments. If you want to test server travel in Editor, ensure that you have "Use Single Process" disabled in "Editor Preferences/Level Editor/Play/Multiplayer Options" and set "Editor Multiplayer Mode" to "Play As Client".
```

File: SpatialGDK/Documentation/unreal-features-support.md (hunk 1 above)
Comment (ernopp) [reply SpatialGDK/Documentation/unreal-features-support.md]: I think the cell type should be "Supported with caveats or workarounds" not supporte

**Comment by MatthewSandfordImprobable** [comment]
@ElleEss Is this all ok?

File: SpatialGDK/Documentation/unreal-features-support.md (hunk 1 above)
Comment (ernopp) [reply SpatialGDK/Documentation/unreal-features-support.md]: Actually maybe this is fine given the whole table is for single-server support only

File: SpatialGDK/Documentation/unreal-features-support.md [hunk 3]
```diff
@@ -319,8 +319,8 @@ Support of Unreal features with the GDK in a single server-worker configuration:
  <tr>
     <td rowspan="3"><a href="https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling"><b>Travel<b></a></td>
     <td>Server Travel</td>
-    <td class="unplanned"></td>
-    <td>Server Travel is on the <a href="https://github.com/spatialos/UnrealGDK/projects/1#card-22461878"> roadmap</a> but is not currently supported.</td>
+    <td class="supported"></td>
+    <td>Not supported when used in conjunction with zoning or offloading</td>
```
Comment (ElleEss) [inline SpatialGDK/Documentation/unreal-features-support.md]: Personally, I would have all of the travels as `ServerTravel` and `ClientTravel` and `SeamlessTravel` to be consistent, but I don't want to change the whole doc here.

Suggest use consistent terms and info.

```suggestion
    <td>Not supported with multiserver zoning or offloading. Multiserver support is on the [roadmap](https://github.com/spatialos/UnrealGDK/projects/1#card-22461878).</td>
```

File: SpatialGDK/Documentation/content/map-travel.md [hunk 4]
```diff
@@ -66,5 +69,6 @@ When launching a client from the SpatialOS Console using the [Launcher](https://
 Please see the [Map Travel URL options]({{urlRoot}}/content/command-line-arguments#map-travel-url-options) section for a list of all URL options that can be used with SpatialOS.
 
 <br/>------<br/>
+_2019-11-26 Page updated with limited editorial review: Added single server Server Travel_
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/map-travel.md]:
```suggestion
_2019-11-26 Page updated with limited editorial review: Added single server Server Travel_ </br>
```

File: SpatialGDK/Documentation/content/map-travel.md [hunk 5]
```diff
@@ -44,7 +44,13 @@ PlayerController->ClientTravel(TravelURL.ToString(), TRAVEL_Absolute, false /*bS
 ```
 
 ## UWorld::ServerTravel
-> Warning: `ServerTravel` is on the [roadmap](https://github.com/spatialos/UnrealGDK/projects/1#card-22461878) but is not currently supported.
+
+> Warning: Multiserver `ServerTravel` is on the [roadmap](https://github.com/spatialos/UnrealGDK/projects/1#card-22461878) but is not currently supported.
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/map-travel.md]:
```suggestion
> Note: Multiserver `ServerTravel` is on the [roadmap](https://github.com/spatialos/UnrealGDK/projects/1#card-22461878) but is not currently supported.
```

**Pull request merged**